pyodide-build/pyodide_build/cli/create_zip.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+import typer  # type: ignore[import]
```

Comment:
It's weird that mypy complains. Maybe it should be added to https://github.com/pyodide/pyodide/blob/main/pyproject.toml#L53?

Reply:
Added typer to [[tools.mypy.overrides]].